### DISCUSSION TITLE
Log in Mecha

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#112] [Q&A]
Bisa enggak, Mas, Mecha dibuat seperti blog privat seperti di Blogger? Kalau kita akses domain dari segala arah di hadapkan ke laman log in terlebih dahulu untuk kemudian memasukkan user name dan password agar bisa akses blog kita.

**Comment by taufik-nurrohman** [comment]
Bisa, cuma belum ada ekstensinya sih untuk sekarang.
